tests/recovery/recovery_test/recovery_test.go

```diff
@@ -0,0 +1,109 @@
+//go:build darwin || (linux && amd64)
+// +build darwin linux,amd64
+
+package recovery
+
+import (
+	"errors"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kopia/kopia/tests/recovery/blobmanipulator"
+	"github.com/kopia/kopia/tests/robustness/snapmeta"
+	"github.com/kopia/kopia/tests/tools/kopiarunner"
+)
+
+func TestSnapshotFix(t *testing.T) {
+	// assumption: the test is run on filesystem and not directly on object store
+	dataRepoPath := path.Join(*repoPathPrefix, dataSubPath)
+
+	baseDir := makeDir("base-dir-")
+
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			log.Println("Skipping robustness tests because KOPIA_EXE is not set")
+		} else {
+			log.Println("Error creating Blob Manipulator:", err)
+		}
```

Comment:
Shouldn't you fail the test right about here?

Reply:
done